backend/module/chaos/serverexperimentation/server_experimentation.go

```diff
@@ -0,0 +1,85 @@
+package serverexperimentation
+
+import (
+	"errors"
+	"fmt"
+
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/uber-go/tally"
+	"go.uber.org/zap"
+
+	experimentation "github.com/lyft/clutch/backend/api/chaos/experimentation/v1"
+	serverexperimentation "github.com/lyft/clutch/backend/api/chaos/serverexperimentation/v1"
+	"github.com/lyft/clutch/backend/module"
+	"github.com/lyft/clutch/backend/service"
+	"github.com/lyft/clutch/backend/service/chaos/experimentation/experimentstore"
+)
+
+const (
+	Name = "clutch.module.chaos.serverexperimentation"
+)
+
+type Service struct {
+	experimentStore experimentstore.ExperimentStore
+}
+
+// New instantiates a Service object.
+func New(_ *any.Any, logger *zap.Logger, scope tally.Scope) (module.Module, error) {
+	store, ok := service.Registry[experimentstore.Name]
+	if !ok {
+		return nil, errors.New("could not find experiment store service")
+	}
+
+	experimentStore, ok := store.(experimentstore.ExperimentStore)
+	if !ok {
+		return nil, errors.New("service was not the correct type")
+	}
+
+	return &Service{
+		experimentStore: experimentStore,
+	}, nil
+}
+
+func (s *Service) Register(r module.Registrar) error {
+	s.experimentStore.RegisterTransformation("type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestConfig", transform)
```

Comment:
register a handler that tells `experimentation` package how to transform configs of `type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestConfig` into the list of `Properties` that can be displayed in experiments table and/or experiment details view.